pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -89,29 +90,40 @@ stdenv.mkDerivation (finalAttrs: {
       ]
     );
 
+  outputs = [ "out" "man" ];
```

Comment:
Phases order wise, I don't know where to put `outputs`. @SuperSandro2000

Reply:
I am more or less following nixpkgs-hammering. Complex attributes (in the sense of "composed by other attributes" - lists, sets &c.) come first.

maybe right after src?